docs/api/navigation-history.md

```diff
@@ -0,0 +1,26 @@
+## Class: NavigationHistory
+
+> Handles session history, maintaining a list of navigation entries for backward and forward navigation.
+Process: [Main](../glossary.md#main-process)<br />
+_This class is not exported from the `'electron'` module. It is only available as a return value of other methods in the Electron API._
+
+### Instance Methods
+
+#### `navigationHistory.getActiveIndex()`
+
+Returns `Integer` - The index from which we would go back/forward or reload.
```

Comment:
Does "from which we would go back/forward or reload" just imply that this is the current page?

Reply:
Yes! Edited for clarity.